_opcodes/op22-bgee.html

```diff
@@ -0,0 +1,38 @@
+---
+n: 22
+name: "Stat: Cumulative Luck Bonus"
+param1: "Statistic Modifier"
+param2: "Type"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies the modifier value specified by the 'Statistic Modifier' field in the style specified by the 'Type' field. Luck affects the minimum damage caused by a weapon (up to the normal maximum damage of the weapon) and the attack roll modifier (up to a maximum of 20, excluding natural attack roll modifiers, e.g. high STR, invisibility etc) by 'Statistic Modifier' amount. 'Statistic Modifier' also affects the damge rolls of opponents (in feature blocks, not extension headers). The effect does not set <a href="../files/ids/bgee/state.htm#BIT26">STATE_LUCK</a>.
+<br/>Known values for 'Type' are:
+<ul>
+	<li>0 &longrightarrow; Cumulative Modifier: Luck = Luck + 'Statistic Modifier' value</li>
+	<li>1 &longrightarrow; Flat Value Modifier: Luck = 'Statistic Modifier' value</li>
+	<li>2 &longrightarrow; Percentage Modifier: Luck = (Luck * 'Statistic Modifier' value) / 100</li>
+</ul>
+
+{% capture note %}
+Effects of Luck:
```

Comment:
This can be merged back to the main file.

Reply:
It also looks like you missed saving throws.

Reply:
> It also looks like you missed saving throws.

Correct, no save bonus – any save bonus has to come from other opcodes.

Reply:
Ah, right, we have the same in GemRB. Maybe they considered it too powerful.